core/src/services/routing.js

```diff
@@ -15,6 +15,26 @@ const getViewUrl = pathData => {
   return lastElement ? lastElement.viewUrl : '';
 };
 
+const getDefaultPathSegment = function(pathData) {
+  const lastElement = [...pathData.navigationPath].pop();
```

Comment:
Even though it is longer, I would use ```const lastElement = pathData.navigationPath[pathData.navigationPath.length - 1]``` because it is more explicit.